### PR TITLE
Poleedrew hw0 attempt1

### PR DESCRIPTION
**Pull request by poleedrew**
---
A good PR should include the following parts:

- [x] A clear title
- [x] A meaningful message for PR, as well as its commits
- [x] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [x] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

**Comment by tigercosmos** [comment]
why do you push an empty file `hw0/countline.py`?

**Comment by poleedrew** [comment]
> why do you push an empty file `hw0/countline.py`?

Sorry, I changed the permission of the file `hw0/countline.py` to make the program executable, but I laterly found out that the one in my username directory was actually executed.

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 This repository is for homework assignments for `Numerical Software Development
 (NYCU 2022 autumn)
-<https://yyc.solvcon.net/en/latest/nsd/schedule/nsd_22au_nycu.html>`__.
+<https://yyc.solvcon.net/en/latest/nsd/schedule/22au_nycu/schedule.html>`__.
```
Comment (yungyuc) [inline README.rst:3]: Could you please avoid including unnecessary changes to your PR?

Comment (poleedrew) [reply README.rst]: Sorry, I didn't notice it.

Comment (yungyuc) [reply README.rst]: No problem, but you didn't resolve the issue.  You included an unnecessary change from me cdb97d4 and it should not be there.  Please drop it from the history.

When you fix that you can probably squash the rest of you change into one.

**Review by yungyuc — APPROVED** [review]
